### PR TITLE
remove jenkins configs and scrips

### PR DESCRIPTION
**Pull request by metaflow**
- moved linux agents to a new node pool (basically renamed old one)

- removed some out of date scripts or moved them to playlists

- removed shell scrips invoked from jenkins

- minor docs updates

Some python classes methods are not used anymore and will be cleaned up
separately.

### CODE REVIEW

**Comment by metaflow** [comment]
@ChristianKuehnel I have tried to run this branch on master / diff changes and it seems to be fine.
Going to add a tag to "just before" removal so it will be easier to find Jenkins scripts just in case.

**Comment by metaflow** [comment]
For #232

File: kubernetes/phabricator-proxy/kustomization.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-namespace: jenkins
+namespace: buildkite
```
Comment (ChristianKuehnel) [inline kubernetes/phabricator-proxy/kustomization.yaml:17]: Nit:alternatively you could also use the `default` namespace, as we're not separating things anyway...

File: kubernetes/reverse-proxy/Ingress.yaml [hunk 2]
```diff
@@ -1,120 +0,0 @@
-# Copyright 2019 Google LLC
```
Comment (ChristianKuehnel) [inline kubernetes/reverse-proxy/Ingress.yaml:1]: Aren't you using the reverse proxy also for the TLS encryption of the Phabricator-Proxy service?

**Review by ChristianKuehnel — APPROVED** [review]
LGTM

Not sure about deleting the nginx reverse proxy. Do you use that for the Phabricator proxy?

File: docs/playbooks.md [hunk 3]
```diff
@@ -1,24 +1,22 @@
...
     + [Buildkite](#buildkite)
-    + [Jenkins](#jenkins)
   * [Testing scripts locally](#testing-scripts-locally)
   * [Custom environment variables](#custom-environment-variables)
   * [Testing changes before merging](#testing-changes-before-merging)
-- [Update HTTP auth credentials](#update-http-auth-credentials)
-- [Phabricator integration](#phabricator-integration)
-  * [Herald](#herald)
-  * [Harbormaster](#harbormaster)
+  * [Update HTTP auth credentials](#update-http-auth-credentials)
```
[the remark this answers is not in the document]
Comment (metaflow) [reply docs/playbooks.md:9]: yes, https://build.llvm-merge-guard.org/ uses HTTP auth

File: kubernetes/phabricator-proxy/kustomization.yaml (hunk 1 above)
Comment (metaflow) [reply kubernetes/phabricator-proxy/kustomization.yaml:17]: agree, it's for historical reasons when we had jenkins and buildkite. On the other hand, having jenkins in a separate workspace made transition a bit easier. Maybe we will need something else and having a buildkite in a separate workspace might help.

File: kubernetes/reverse-proxy/Ingress.yaml (hunk 2 above)
Comment (metaflow) [reply kubernetes/reverse-proxy/Ingress.yaml:1]: aha, I have moved cert manager setup to an upper level as it was all that left.

**Comment by metaflow** [comment]
> Not sure about deleting the nginx reverse proxy. Do you use that for the Phabricator proxy?

Yes, we still use nginx for https://build.llvm-merge-guard.org/ though ngins setup is done following standard instructions and the only config we had is is for let's encrypt.

**Comment by metaflow** [comment]
@ChristianKuehnel  FYI: I have found that currently running nginx instance still uses jenkins-agents nodepool. Please don't remove it yet.

**Pull request merged**